stack.yaml

```
@@ -4,15 +4,12 @@ packages:
 extra-deps:
   - git: https://github.com/striot/striot.git
     commit: 1441100588615723481b9d7571dbb5b37bbd65d4
-  - algebraic-graphs-0.5
   - aeson-1.4.6.0
```

```suggestion
  - aeson-1.4.6.0
  - network-3.1.0.0
  - unagi-chan-0.4.1.0
  - time-compat-1.9.2.2
  - websockets-0.12.6.1
  - connection-0.3.1
  - socks-0.6.1
```
If you add these extra dependencies and change the snapshot to `lts-13.25` at all seems to build!